### ISSUE TITLE
Why it's loss = loss/(4*d*4) ?

### ISSUE THREAD

**Issue by jindongwang** [SSARCandy/DeepCORAL#14]
I noticed in the Deep CORAL paper that the CORAL loss is calculated by
`loss = loss / (4 * d * d)`
But line 26 of models.py defines the CORAL loss as
`loss = loss/(4*d*4)`
I do not understand. Is there a mistake?

**Comment by SSARCandy** [comment]
you are right, I fixed it. Thanks
